packages/core/src/view/handler/CellEditorHandler.ts

```diff
@@ -882,7 +881,7 @@ class CellEditorHandler implements GraphPlugin {
           }
         });
       }
-
+      this.trigger = null;
```

Comment:
**question:** do you know why this statement must be moved?
In mxGraph, it was at the former place and it worked: https://github.com/jgraph/mxgraph/blob/v4.2.2/javascript/src/js/view/mxCellEditor.js#L968

Reply:
Okay, so this trigger is inside stopEditing function. We change the label inside the if statement if innerHTML is not equal to initial which is okay and we call the applyValue function which takes state and the value, which also seemed fine. 
Problem arised because the applyValue function used this.trigger when calling labelChanged function and labelChanged always received null. This is because no matter what, we set the trigger to null even before checking if values has changed. I can look up logic inside the mxGraph for this aswell, but I since the editing of value doesn't work in the example, I would assume they had the bug too.

Reply:
OK thanks for the information I got what you mean 😄 

I didn't understand why this change was needed at a first glance, because most stories and examples work without this change.
The Editing story relies on the triggered event in the overridden `Graph.labelChanged` method. It needs the event to be correctly set to correctly assign the label (using the custom `Cell.value` which is an object and not a string, which stores the 2 label values). See https://github.com/maxGraph/maxGraph/blob/v0.13.1/packages/html/stories/Editing.stories.js#L116
The regular `Graph.labelChanged` method doesn't directly used the event (except to send another event which is not used in stories). That is why, most of the time, the existing code was working.

For the record, the mxGraph "Editing" example doesn't work: https://jgraph.github.io/mxgraph/javascript/examples/editing.html. It doesn't set the label correctly after edition.